File: TrackEdize/ClientApp/src/app/tracking/models/issue.ts
import { Dropdown } from "src/app/shared/model/dropdown";
import {Comment} from './comment';
import { Sprint } from "./sprint";

export class Issue {
    id?: string;
    subject: string | null = null;
    userName: string | null = null;
    password: string | null = null;
    steps: string | null = null;
    actualResult: string | null = null;
    expectedResult: string | null = null;
    environment: string | null = null;
    version: string | null = null;
    browser: string | null = null;
    device: string | null = null;
    system: string | null = null;
    project: Dropdown = new Dropdown();
    key: string | undefined;
    status: string | undefined;
    type: string | undefined;
    acceptance?: string;
    createdDate?: Date;
    updatedDate?: Date;
    createdBy?: string
    dev?: string;
    qa?:string;
    comments: Comment[] = [];
    startDate: Date | null = null;
    endDate: Date | null = null;
    sprint: Sprint | null = null;

    constructor() {
        
    }
}

File: Database/Entities/Issue.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Database.Entities.Base;
using MongoDB.Bson.Serialization.Attributes;

namespace Database.Entities
{
    public class Issue : BaseEntity
    {
        public int OrderNumber { get; set; }
        public string? Subject { get; set; }
        public string? UserName { get; set; }
        public string? Password { get; set; }
        public string? Steps { get; set; }
        public string? ActualResult { get; set; }
        public string? ExpectedResult { get; set; }
        public string? Environment { get; set; }
        public string? Version { get; set; }
        public string? Browser { get; set; }
        public string? Device { get; set; }
        public string? System { get; set; }
        public Dropdown? Project { get; set; }
        public string? Key { get; set; }
        public string? Status { get; set; }
        public string? Type { get; set; }
        public List<Comment>? Comments { get; set; }
        public string? Acceptance { get; set; }
        public string? CreatedBy { get; set; }


    }
}
